assets/src/edit-story/utils/contrastUtils.js

```diff
@@ -48,14 +48,19 @@ export function calculateLuminanceFromStyleColor(styleColor) {
   return hues.relativeLuminance(rgb);
 }
 
+/**
+ * 18 point text or 14 point bold text is judged to be large enough to require a lower contrast ratio.
+ * https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html
+ */
 /**
  * Check contrast ratios from luminances for WCAG guidelines
  *
  * @param  {number} luminanceA Luminance A
  * @param  {number} luminanceB Luminance B
+ * @param  {number} fontSize Font size
  * @return {Object} WCAG contrast ratio checks
  */
```

Comment:
This could be easily part of the docblock.

```suggestion
/**
 * Check contrast ratios from luminances for WCAG guidelines.
 *
 * 18 point text or 14 point bold text is judged to be large enough to require a lower contrast ratio.
 * See https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html
 *
 * @param  {number} luminanceA Luminance A
 * @param  {number} luminanceB Luminance B
 * @param  {number} fontSize Font size
 * @return {Object} WCAG contrast ratio checks
 */
```